src/config.rs

```diff
@@ -8,7 +8,6 @@ use strum_macros::{Display, EnumString};
 #[derive(Serialize, Deserialize, Debug)]
 pub struct Location {
     pub name: String,
```

Comment:
could possibly do away with this struct altogether if it's just a name?

Reply:
I think eventually we'll end up using some of the other fields if we ever want to implement pull etc (so we'd need type and args). A struct with just a string should be optimised down to just a string, so it doesn't cost much